coveralls.json

```diff
@@ -0,0 +1,12 @@
+{
+  "coverage_options": {
+    "minimum_coverage": 100
```

Comment:
[This dependency](https://github.com/parroty/excoveralls) just names that as the configuration json file, but you can use codecov with this and arn't bound to coveralls